extensions/lifecycle/LifecycleMetrics.js

```diff
@@ -1,4 +1,5 @@
 const { ZenkoMetrics } = require('arsenal').metrics;
+const { getStatusLabel } = require('../utils/metricsUtil');
 
 const LIFECYCLE_LABEL_ORIGIN =  'origin';
 const LIFECYCLE_LABEL_OP = 'op';
```

Comment:
It would be useful to have "duration" metrics as well: e.g.
* the duration of individual tasks (processing of an object by GC...)
* duration of bucket scan
* latency/delay between an "event" (expiration....) and the time when we actually apply the transition

Reply:
This deserves a ticket for itself: https://scality.atlassian.net/browse/BB-343